gtk3-macros/Cargo.toml

```
@@ -25,3 +26,6 @@ proc-macro2 = "1.0"
 quote = "1.0"
 syn = { version = "1.0", default-features = false, features = ["full"] }
 proc-macro-crate = "1.0"
+
+[dev-dependencies]
+gtk = { path = "../gtk" }
```

```suggestion
gtk = { path = "../gtk" }
glib = { path = "../glib" }
```

So that this is fixed probably

```
 error[E0433]: failed to resolve: use of undeclared crate or module `glib`
  --> src/lib.rs:45:7
   |
19 |     #[glib::object_subclass]
   |       ^^^^ use of undeclared crate or module `glib`
```